source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -0,0 +1,628 @@
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Description
+*************
+
+Telemetry, one of the key features of |CL|, enables developers to observe and proactively address issues in the OS before end users are impacted.
+
+Telemetrics is a combination word made from:
+
+* Telemetry, which is sensing and reporting data.
+* Analytics, which is using visualization and statistical inferencing to make sense of the reported data.
+
+|CL| telemetry reports system-level debug/crash information using specialized probes. The probes monitor system tasks such as swupd, kernel oops, machine error checks, and the BIOS error report table for unhandled hardware failures. Telemetry enables real-time issue reporting to allow system developers to quickly focus on an issue and monitor corrective actions.
+
+|CL| telemetry is fully customizable and can be used during software development for debugging purposes. You can use the  libtelemetry library in your code to create custom telemetry records. You can also use the telem-record-gen utility in script files for light touch record creation where instrumenting code files doesn't make sense.
+
+Architecture
+============
+
+|CL| telemetry has two fundamental components, which are shown in figure 1:
+
+* Client: generates and delivers records to the backend server via the network
+
+* Backend: receives records sent from the client and displays the cumulative content through a specialized web interface.
+
+.. figure:: ../../concepts/figures/telemetry-about-1.png
+   :alt: Figure 1, Telemetry Architecture
+
+   Figure 1: :guilabel:`|CL| Telemetry Architecture`
+
+The telemetry client provides the front end of the telemetrics solution and includes the following components:
+
+* telemprobd, a daemon that receives and prepares telemetry records from probes and spools them to disk.
+* telempostd, a daemon that manages spooled telemetry records and delivers these records according to configurable settings.
+* probes, that collect specific types of data from the operating system.
+* libtelemetry, the API that telemetrics probes use to create records.
+
+The telemetry backend provides the server-side component of the telemetrics solution and consists of:
+
+* Nginx web server.
+* Two Flask apps:
+  * Collector, an ingestion web app for records received from client probes.
+  * TelemetryUI, a web app that exposes different views to visualize the telemetry data.
+* PostgreSQL as the underlying database server.
+
+.. note::
+
+   The default telemetry backend server is hosted by the Intel |CL| development team and is not viewable outside the Intel firewall. To collect your own records, you must set up your own telemetry backend server.
+
+.. note::
+
+   The telemetry functionality adheres to `Intel privacy policies`_ regarding the collection and use of :abbr:`PII (Personally Identifiable Information)` and is open source.
```

Comment:
Please add a URL to which `Intel privacy policies`_ refers at bottom of page. Required for this PR to build.

Reply:
Changes made.  Can we meet before merging to discuss images/paths &c

Reply:
Of course. Just send me a meeting invitation for Wed.